app/src/main/java/co/condorlabs/customcomponents/customedittext/EditTextCurrencyField.kt

```diff
@@ -48,7 +49,16 @@ class EditTextCurrencyField(context: Context, attrs: AttributeSet) : BaseEditTex
         }
     }
 
+    override fun onFocusChanged(focused: Boolean, direction: Int, previouslyFocusedRect: Rect?) {
+        super.onFocusChanged(focused, direction, previouslyFocusedRect)
+        if (focused) {
+            if (editText?.text.isNullOrEmpty()) {
+                editText?.setText(DOLLAR_SYMBOL)
+            }
+        }
+    }
+
     private fun isFieldEmpty(text: String?): Boolean {
-        return text == DOLLAR_SYMBOL
+        return text == DOLLAR_SYMBOL || text == EMPTY
```

Comment:
Then rename the function: `isNotFilledWithDigits`

Reply:
done